azure-pipelines.yml

```diff
@@ -16,7 +16,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
+trigger:
+    branches:
+        exclude:
+        - dependabot/*
```

Comment:
very very minor comment about indent :) missing space here

Reply:
@pascalgrimaud : Which line are you referring to here? :slightly_smiling_face: